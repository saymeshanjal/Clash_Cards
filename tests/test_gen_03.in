10
2 48 1
2 34 3
2 14 2
6 26 4
16 8 3
6 30 5
16 20 3
16 22 4
10 7 5
16 34 2