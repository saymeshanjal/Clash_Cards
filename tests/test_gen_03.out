1 1 0
0 2 0
0 1 1
1 14 5
1197 11673 0
1 19 0
1744 11126 0
169 10765 1936
0 252 0
254 1011 11605
